starknet_devnet/devnet_overrides.py

```diff
@@ -0,0 +1,524 @@
+import sys
```

Comment:
If this file contains only overrides of rust-vm, then I think `devnet_overrides.py` is a too broad name. In fact, why not put the content of this file into `cairo_rs_py_patch.py`?

Reply:
Good idea, I may have went too far with the file abstraction. Thanks Fabijan